src/Aquifer.Well.API/Program.cs

```diff
@@ -50,6 +52,11 @@
 builder.Services.AddOptions<ConfigurationOptions>().Bind(builder.Configuration);
 builder.Services.Configure<ApiKeyAuthorizationMiddlewareOptions>(o => o.Scope = ApiKeyScope.WellApi);
 
+builder.Services.AddSingleton<INotificationService, NotificationHubService>();
+builder.Services.AddOptions<NotificationHubOptions>()
```

Comment:
This shouldn't be necessary if you add a `NotificationHub` property to the `ConfigurationOptions` class which is already bound above.